jupyterlab/labbox_ephys_widgets_jp/labbox_ephys_widgets_jp/create_recording_view.py

```diff
@@ -50,15 +49,15 @@ def _handle_message(self, widget, msg, buffers):
                 try:
                     job: HitherJob = hi.run(functionName, **kwargs, labbox=labbox_context).wait()
                 except Exception as err:
-                    self.send(dict(type='test 3'))
                     self.send({
                         'type': 'hitherJobCreationError',
                         'client_job_id': client_job_id,
                         'error': str(err) + ' (new method)'
                     })
                     return
-                self.send(dict(type='test 4'))
-                setattr(job, '_client_job_id', client_job_id)
+                client_job_ids = getattr(job, '_client_job_ids', [])
```

Comment:
attach more than one client job id to the same hither job

Reply:
So this is basically going to mean that multiple computations can be pushed to the remote server with one hither call, is that right?

Reply:
Well. It means that we can handle the situation where multiple client-side jobs correspond to the same server-side job. Actually don't know if this happens in practice. If it did happen, then it would be because the identical job was requested multiple times prior to the job completing. But again I don't know if it actually happens. I was having some trouble and thought this might be the cause.